lookout/style/format/feature_utils.py

```diff
@@ -132,14 +132,15 @@ def from_node(node: bblfsh.Node, file: str, path: str,
 EMPTY_CLS = frozenset([CLASS_INDEX[CLS_TAB_DEC], CLASS_INDEX[CLS_SPACE_DEC],
                        CLASS_INDEX[CLS_NOOP]])
 CLS_TO_STR = {
-    CLS_DOUBLE_QUOTE: '"',
-    CLS_NEWLINE: "\n",
-    CLS_NOOP: "",
-    CLS_SINGLE_QUOTE: "'",
     CLS_SPACE: " ",
-    CLS_SPACE_DEC: "",
-    CLS_SPACE_INC: " ",
     CLS_TAB: "\t",
-    CLS_TAB_DEC: "",
+    CLS_NEWLINE: "\n",
+    CLS_SPACE_INC: " ",
+    CLS_SPACE_DEC: "",
     CLS_TAB_INC: "\t",
+    CLS_TAB_DEC: "",
+    CLS_SINGLE_QUOTE: "'",
+    CLS_DOUBLE_QUOTE: '"',
+    CLS_NOOP: "",
 }
+INDEX_CLS_TO_STR = {i: c for i, c in enumerate(CLS_TO_STR)}
```

Comment:
This is a bad thing to do. The order of dictionary keys is not constant in 3.5, and anyway it's a bad idea to depend on it. Why do you need this anyway? You can write

```python
INDEX_CLS_TO_STR = tuple(CLS_TO_STR(c) for c in CLASSES)
```